dev-lang/nasm/nasm-2.14.02.recipe

```diff
@@ -0,0 +1,59 @@
+SUMMARY="The NASM assembler"
+DESCRIPTION="The Netwide Assembler, NASM, is an 80x86 and x86-64 assembler \
+designed for portability and modularity. It supports a range of object file \
+formats, including Linux and *BSD a.out, ELF, COFF, Mach-O, Microsoft 16-bit \
+OBJ, Win32 and Win64. It will also output plain binary files. Its syntax is \
+designed to be simple and easy to understand, similar to Intel's but less \
+complex. It supports all currently known x86 architectural extensions, and \
+has strong support for macros."
+HOMEPAGE="https://www.nasm.us/"
+COPYRIGHT="1996-2018 The NASM Authors"
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="https://www.nasm.us/pub/nasm/releasebuilds/$portVersion/nasm-$portVersion.tar.bz2"
+CHECKSUM_SHA256="34fd26c70a277a9fdd54cb5ecf389badedaf48047b269d1008fbc819b24e80bc"
+PATCHES="nasm-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	nasm$secondaryArchSuffix = $portVersion compat >= 1
+	cmd:nasm = $portVersion compat >= 1
+	cmd:ndisasm = $portVersion compat >= 1
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku_devel$secondaryArchSuffix
```

Comment:
Correction:
```diff
- 	haiku_devel$secondaryArchSuffix
+ 	haiku${secondaryArchSuffix}_devel
```

Reply:
Fixed this.
So package suffixes always go last?